README.md

```diff
@@ -32,6 +32,10 @@ The Fluid community is an international group of designers, developers, and test
 
 The best way to join the Fluid Community is to jump into any of our community activities. Visit our [website](http://fluidproject.org/) for links to our mailing lists, chat room, wiki, etc.
 
+## Inclusion ##
+
+The Fluid community is dedicated to inclusive design&mdash;design that considers the full range of human diversity with respect to ability, language, culture, gender, age and other forms of human difference. To help ensure that our community is a safe space for all contributors, we have adopted a [code of conduct](https://wiki.fluidproject.org/display/fluid/Inclusion+in+the+Fluid+Community) based on the Contributor Covenant. All participants and contributors have the responsibility to uphold this code. Please contact the [Advocacy working group](mailto:fluid-advocacy@fluidproject.org) if you encounter unacceptable behaviour.
```

Comment:
I've added a link to the Contributor Covenant website. I think it's important for people to see which CoC we're using. One of the issues identified during our discussions of CoCs was the value of choosing a recognizable, established code. Mentioning it specifically helps with recognition.